src/configs/test.yml

```diff
@@ -27,6 +27,15 @@ tables:
     max-mb: 10000
     max-hr: 240
     schema: "ROW<id:int, event:string, items:list<string>, flag:bool, value:tinyint>"
+    transformer:
+      schema: "ROW<id:int, event:string, item:string, flag:bool, value:tinyint, ts:long>"
```

Comment:
Just to keep on the same page, Nebula supports compound types https://github.com/varchar-io/nebula/blob/master/src/type/Type.h#L43-L45, it's just we have bugs to fix to make them work end-to-end.